### DISCUSSION TITLE
False positive Person detected

### DISCUSSION THREAD

**Discussion by lpdescamps** [TheRealWaldo/AMG8833-ESPHOME#12] [Q&A]
Hi,

thanks for that great project!

I just can't figure out how to fix those false positive person detected.
It detects a person if I aim the sensor to 
- my log burner
- my dog
- my underfloor heating

what am I doing wrong?
below my yaml config.
many thanks

[esp-frtdoorthermsens.yaml.txt](https://github.com/TheRealWaldo/AMG8833-ESPHOME/files/7764245/esp-frtdoorthermsens.yaml.txt)

**Comment by TheRealWaldo** [comment]
The rudimentary detection works off of a difference in temperature.  Those things would be quite a bit hotter than your ambient temperature, and would trigger it.

Careful aim is the best way to avoid that situation, otherwise you'll need to mask the pixels around those areas.  This would require modifying the source code to ignore the pixels in question.
